src/pages/simulator/recurring/index.tsx

```diff
@@ -50,13 +50,17 @@ export const SimulatorInputRecurringPage = () => {
       const multiplierMax = type === 'buy' ? 0.1 : 0.2;
       const multiplierMin = type === 'buy' ? 0.2 : 0.1;
 
-      const max = new SafeDecimal(marketPrice)
-        [operation](marketPrice * multiplierMax)
-        .toFixed();
-
-      const min = new SafeDecimal(marketPrice)
-        [operation](marketPrice * multiplierMin)
-        .toFixed();
+      const max = marketPrice
+        ? new SafeDecimal(marketPrice)
+            [operation](marketPrice * multiplierMax)
+            .toFixed()
+        : '';
+
+      const min = marketPrice
+        ? new SafeDecimal(marketPrice)
+            [operation](marketPrice * multiplierMin)
+            .toFixed()
+        : '';
```

Comment:
This is very difficult to read, I think it is worth refactoring a little.

Reply:
Refactored